docs/ui/components/Footer/NewsletterSignUp.tsx

```diff
@@ -0,0 +1,92 @@
+import { css } from '@emotion/react';
+import { borderRadius, shadows, spacing, theme, typography } from '@expo/styleguide';
+import { useState } from 'react';
+
+import { Button } from '../Button';
+import { A, DEMI, P } from '../Text';
+
+export const NewsletterSignUp = () => {
+  const [email, setEmail] = useState('');
+  const [userSignedUp, setUserSignedUp] = useState(false);
+
+  return (
+    <div css={wrapperStyle}>
+      <DEMI theme="secondary" css={textStyle}>
+        Sign up for developer updates
+      </DEMI>
```

Comment:
```suggestion
      <CALLOUT theme="secondary" weight="medium">
        Sign up for developer updates
      </CALLOUT>
```
We can use the `<CALLOUT>` element, instead of manually defining type styles.